components/CodeBlock.tsx

```diff
@@ -132,6 +142,15 @@ export function RawCodeBlock({
           );
         })}
       </div>
+      {enableCopyButton &&
+      (
+        <button
+          className={tw`opacity-0 group-hover:opacity-100 rounded border border-[#D2D2DC] p-1.5 self-start`}
+          onClick={`navigator?.clipboard?.writeText('${code.trim()}');`}
```

Comment:
I'm getting a type error here, but if I add a `// @ts-ignore` deno panics when running `deno fmt`, not sure if I can work around that.

Reply:
thats known issue. dprint/dprint-plugin-typescript/issues/400. add a format ignore at the top of the file